README.md

```diff
@@ -39,7 +39,19 @@ However, the following temporary limitations are presently in place:
 - Only works with Google Cloud SQL and AWS RDS PostgreSQL.
 
 
-## 🚀 Installation
+## 🚀 Quick Installation
+
+First, you need to apply the prerequisites from the [Prerequisites](#prerequisites) section below.
```

Comment:
I would suggest flipping this around: include the prerequisites in the Quick Installation and reference it from the Detailed Installation.

Reply:
Thanks. Applied your suggestion, but without back referencing to quick installation from detailed installation.